src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationRefreshOptions.cs

```diff
@@ -55,15 +55,27 @@ public AzureAppConfigurationRefreshOptions Register(string key, string label = L
         /// Any refresh operation triggered using <see cref="IConfigurationRefresher"/> will not update the value for a key until the cached value for that key has expired.
         /// </summary>
         /// <param name="cacheExpiration">Minimum time that must elapse before the cache is expired.</param>
+        [Obsolete("The SetCacheExpiration method will be deprecated in a future release. Please use the `SetRefreshInterval` method instead. " +
+            "Note that only the name of the method has changed, and the functionality remains the same.")]
         public AzureAppConfigurationRefreshOptions SetCacheExpiration(TimeSpan cacheExpiration)
         {
-            if (cacheExpiration < RefreshConstants.MinimumCacheExpirationInterval)
+            return SetRefreshInterval(cacheExpiration);
+        }
+
+        /// <summary>
+        /// Sets the minimum time interval between consecutive refresh operations for the registered key-values. Default value is 30 seconds. Must be greater than 1 second.
+        /// Any refresh operation triggered using <see cref="IConfigurationRefresher"/> will not update the value for a key unless the refresh interval has elapsed since the key was last refreshed.
```

Comment:
```suggestion
        /// Refresh operation triggered using <see cref="IConfigurationRefresher"/> will not make any server requests unless the refresh interval has elapsed since the key was last refreshed.
```